packages/cashier/src/pages/payment-agent/payment-agent-container/payment-agent-container.jsx

```diff
@@ -173,7 +182,7 @@ const PaymentAgentContainer = ({
             )}
         </React.Fragment>
     );
-};
+});
 
 PaymentAgentContainer.propTypes = {
```

Comment:
@george-usynin-binary These `propTypes` can also be removed except `is_deposit` and `verification_code`.

Reply:
@farzin-deriv oh, sure, thanks for the notice 🙏🏻